packages/devtools-network-console/public/index.html

```diff
@@ -111,6 +111,12 @@
 .request-title-editor > .ms-TextField-wrapper > .ms-TextField-fieldGroup > input.ms-TextField-field::placeholder {
   font-size: 16px !important;
 }
+
+/** https://github.com/microsoft/fast/issues/2235 */
+.content-type-select > div[role=listbox] {
+  position: relative;
+  z-index: 100;
+}
```

Comment:
@robpaveza Would you like a safer way to apply this? It's possible that with react's non-deterministic rendering this may not always apply. We could apply this at the component and have a custom stylesheet applied with greater specificity. Let me know :).

Reply:
Agree on not using arbitrary values - at the component level it's super hard to know what values are being used somewhere like devtools. I've seen apps set certain layers to 10000 because at the time it was at the very top of the stacking context. I prefer systematic approaches but that ends up being overly prescriptive on our end. Either way, there's a method to apply this in a jss stylesheet that will likely be safer. I can advise if a solution like that is preferred.

We're also looking forward to having the magic to stack on top of everything else exposed by the web platform. Once we have that this issue could disappear :).